packages/frontend/src/hooks/useSpaces.ts

```diff
@@ -65,9 +63,6 @@ export const useDeleteMutation = (projectUuid: string) => {
             mutationKey: ['space_delete', projectUuid],
             onSuccess: async () => {
                 await queryClient.refetchQueries(['spaces', projectUuid]);
-                history.push({
-                    pathname: `/projects/${projectUuid}/home`,
-                });
```

Comment:
does this affect any other place?

Reply:
Yes, for example, deleting a space from `all spaces` but in that case, we don't want to redirect (we stay in the same page) . We only redirect if we're deleting on the same `space` page